.github/workflows/ci.yml

```diff
@@ -68,10 +73,9 @@ jobs:
       - name: Vendor Dependencies
         run: make vendor vendor.check
 
-      # This action uses its own setup-go, which always seems to use the latest
-      # stable version of Go. We could run 'make lint' to ensure our desired Go
-      # version, but we prefer this action because it leaves 'annotations' (i.e.
-      # it comments on PRs to point out linter violations).
```

Comment:
Is the part of this comment that was removed no longer true, or was I mistaken when I wrote it?

Reply:
@negz it is still true, but we are now disabling it with `skip-go-installation: true` (which I just added).

Reply:
Ah, got it. It seems like that option didn't exist back when I wrote this comment (it was added inhttps://github.com/golangci/golangci-lint-action/pull/144).